src/meshapi/tasks.py

```diff
@@ -12,21 +13,61 @@
 )
 from meshapi.views.panoramas import sync_github_panoramas
 from meshdb.celery import app as celery_app
+from meshdb.settings import DBBACKUP_STORAGE_OPTIONS
 
 
 @celery_app.task
-def run_database_backup() -> None:
-    logging.info("Running database backup task")
+def run_database_backup() -> bool:
+    # Don't run a backup unless it's prod1
+    environment = f"\"{os.environ.get('MESHDB_ENVIRONMENT', '')}\""
```

Comment:
Should we make `MESHDB_ENVIRONMENT` a variable in `settings.py`?

Reply:
Not a bad idea. Definitely less verbose. What if we do that for all of our environment variables?

Reply:
Done. Not going to do it for all env vars in this PR but that would be an ez first issue lol

Reply:
https://github.com/nycmeshnet/meshdb/issues/515